rally/aci_plugins/neutron/scenarios/extensions/address_scope.py

```diff
@@ -0,0 +1,39 @@
+from rally import consts
+from rally import exceptions
+from rally.common import validation
+from rally.aci_plugins import vcpe_utils
+from rally.aci_plugins import create_ostack_resources
+from rally.plugins.openstack import scenario
+from rally.plugins.openstack.scenarios.nova import utils as nova_utils
+from rally.plugins.openstack.scenarios.neutron import utils as neutron_utils
+import time
+
+@validation.add("required_services", services=[consts.Service.NOVA, consts.Service.NEUTRON])
+@validation.add("required_platform", platform="openstack", users=True)
+@scenario.configure(name="ScenarioPlugin.address_scope", context={"cleanup@openstack": ["nova", "neutron"],
+                             "keypair@openstack": {},
+                             "allow_ssh@openstack": None}, platform="openstack")
+
+class SVIBGPConnectivity(create_ostack_resources.CreateOstackResources, vcpe_utils.vCPEScenario, neutron_utils.NeutronScenario,
+                         nova_utils.NovaScenario, scenario.OpenStackScenario):
+
+   def run(self, cidr1, cidr2, image, flavor, public_net, aci_nodes, username, password):
+
+        try:
+            net1, sub1 = self._create_network_and_subnets({"provider:network_type": "vlan", "apic:nested_domain_name": "dntest1"},{"cidr": cidr1}, 1, None)
```

Comment:
Just curious - why did you opt for "vlan" over "opflex"? Was this required?

Reply:
Also - it's not a big issue, but you don't need the nested_domain_name here (since you're not testing that).